File: 1_Array and Strings/1_1.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author DanielX
 * This System will check user's status either student or teacher
 * Return Student (Or Teacher) Number and Corresponding Courses
 */
import java.io.*;
import java.util.Scanner;

    public boolean isUnique(String str)
        {
            if (str.length() > 128) return false;

            boolean[]char_set = new boolean[256];

            for (int i = 0; i < s.size(); i++)
        {

            int val = str.charAt(i);
            if (char_set[val]){
                return false;
            }
            char_set[val] = true;
        }
        return true;
    }
        }
    public static void main(String[] args) {
    String s = "Test string" ;
    boolean test = isUnique(s);
    System.out.println(test);
       
}

File: 1_Array and Strings/1_1.cpp
#include <iostream>
#include <string>

using namespace std;

bool unique_character(string &s)
	{

		if (s.size() > 128) return false;

		bool char_set [256];
		for (int i = 0; i < s.size(); i++)
		{

			int val = s.at(i);
			char c = s.at(i);
			//cout << c << endl;
			if (char_set[val]){
				cout << "fk" << endl;
				return false;
			}
			cout << "jj" << endl;
			char_set[val] = true;
		}
		return true;
	}

int main()
{
	string s = "Test string" ;
	bool test = unique_character(s);
	cout << test << endl;

}